src/conversion/azure/contracts.ts

```diff
@@ -1,6 +1,6 @@
+import {IEntityStore, NodeSpec} from '../..';
+
 import {AnyAzureObject} from '.';
-import {IEntityStore, IRules} from '..';
-import {NodeSpec} from '../..';
 
 export interface ItemMoniker {
```

Comment:
Can we use the "alias" terminology, instead of "moniker"?

Reply:
As we discussed in our review. I'll leave this a moniker and fixup where the word alias was still being used. We both think we may be able to remove the idea of aliasing/naming in a future change